app/forms/openid_connect_authorize_form.rb

```diff
@@ -129,6 +134,10 @@ def check_for_unauthorized_scope(params)
 
   def parse_to_values(param_value, possible_values)
     return [] if param_value.blank?
+    Rails.logger.debug('parsevalues******************')
+Rails.logger.debug(param_value)
+Rails.logger.debug(possible_values)
+Rails.logger.debug('******************')
```

Comment:
just wanted to flag we should make sure to remove this before merging

Reply:
Yeah, my debugging stuff always sticks in the between commits, but I have a mental task to pull them before finalizing.